index.html

```diff
@@ -8,6 +8,36 @@
   </head>
   <body class="page">
     <div class="page__container">
+      <section class="advantages"> 
+        <div class="advantages__preview"><p class="advantages__heading">Устали от серости мегаполиса?</p><p class="advantages__heading-text">Нам есть, что вам<br> предложить!</p></div>
+        <div class="advantages__container">
+          <div class="advantages__pictures">
+              <div class="advantages__wrapper"><img src="images/advantages-iphone-grey-desktop.png" alt="Телефон с серым изображением мусорников" class="advantages__picture advantages__picture_hidden"></div>
+              <div class="advantages__wrapper"><img src="images/advantages-iphone-pink-desktop.png" alt="Телефон с розовым изображением мусорников" class="advantages__picture advantages__picture_hidden"></div>
+              <div class="advantages__wrapper">
+                <picture>
+                  <source srcset="images/advantages-iphone-smiler-tablet.png" media="(max-width: 1199px)">
+                  <source srcset="images/advantages-iphone-smiler-mobile.png" media="(max-width: 659px)">
+                  <img src="images/advantages-iphone-smiler-desktop.png" alt="Телефон с изображением улыбающегося смайлика" class="advantages__picture">
+                </picture>
+              </div>
+          </div>
+          <ul class="advantages__list">
+            <li class="advantages__info">
```

Comment:
@NazarPauk, what do you think about renaming `advantages__info` to `advantages__list-item`? 

This change would make the naming a bit more abstract. Considering our section structure so far, we don't need to specify that it's "info" since it's clear that we're dealing with items in a list of advantages. This could also make it easier to maintain our app in case the content changes, without needing to rename the list items.

html
```html
<section class="advantages">
  <div class="advantages__intro">...</div>
  <div class="advantages__content">  
    <ul class="advantages__list">
      <li class="advantages__list-item">...</li>
      <!-- other list items-->
    </ul>
  </div>
</section>
```

Reply:
Thank you for your comments and advice; renaming the classes really simplified the understanding of the code.❤️